shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -180,3 +180,39 @@ subquery
 lockClause
     : FOR UPDATE 
     ;
+
+merge
+    : MERGE hint? intoClause usingClause mergeUpdateClause? mergeInsertClause? errorLoggingClause?
+    ;
```

Comment:
@ThanoshanMV You should add the rule to `shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/org/apache/shardingsphere/sql/parser/autogen/OracleStatement.g4`.

Reply:
When you add the rule to `OracleStatement.g4`, maybe resolve the follow issue.
```
line 1:0 no viable alternative at input 'MERGE'
```

Reply:
Hi @jingshanglu, yes it solved that issue